resources/scss/componets/11_pricing.scss

```diff
@@ -0,0 +1,130 @@
+        border-radius: 5px;
+        position: relative;
+        @media screen and (max-width: 991px) {
+            margin-bottom: 50px;
+        }
+
+        .popular {
+            position: absolute;
+            width: 80%;
+            text-align: center;
+            height: 40px;
+            line-height: 40px;
+            background: linear-gradient(45deg, $theme 0%, $yellow 100%);
+            color: $white;
+            top: -20px;
+            left: 10%;
+
+            &:before {
+                border-right: 20px solid transparent;
+                left: -1px;
+                content: '';
+                width: 0;
+                height: 0;
+                border-bottom: 40px solid $white;
+                position: absolute;
+                top: 1px;
+            }
+
+            &:after {
+                border-left: 20px solid transparent;
+                right: -1px;
+                content: '';
+                width: 0;
+                height: 0;
+                border-bottom: 40px solid $white;
+                position: absolute;
+                top: 1px;
+            }
+
+            span {
+                position: absolute;
+                width: 100%;
+                height: 100%;
+                top: 0;
+                left: 0;
+
+                &:before {
+                    border-bottom: 20px solid #c00f4a;
+                    border-right: 11px solid transparent;
+                    left: 0;
+                    content: '';
+                    width: 0;
+                    height: 0;
+                    position: absolute;
+                    top: 0;
+                    z-index: 2;
+                    opacity: .7;
+                }
+
+                &:after {
+                    border-bottom: 20px solid #d6961e;
+                    border-left: 11px solid transparent;
+                    right: 0;
+                    content: '';
+                    width: 0;
+                    height: 0;
+                    position: absolute;
+                    top: 0;
+                    z-index: 2;
+                    opacity: .7;
+                }
+            }
+        }
+
+        .type {
+            padding-bottom: 30px;
+            margin-bottom: 30px;
+            border-bottom: 1px solid $text;
+            position: relative;
+
+            h6 {
+                font-size: 20px;
+                font-weight: 500;
+            }
+
+            &:after {
+                content: '';
+                width: 20px;
+                height: 20px;
+                border-right: 1px solid $text;
+                border-bottom: 1px solid $text;
+                background: $white;
+                position: absolute;
+                bottom: -11px;
+                left: calc(50% - 10px);
+                transform: rotate(45deg);
+            }
+        }
+
+        .price {
+            margin: 30px 0;
+
+            h4 {
```

Comment:
do not use direct element tag in css
set class name like **.title** , **.sub-title** , **.slogal** , **.description** etc. like these

Reply:
Fixed